tensorflow/contrib/lite/examples/ios/simple/RunModelViewController.mm

```diff
@@ -118,52 +113,51 @@ - (UIImage *)fixrotation:(UIImage *)image{
     
     // Now we draw the underlying CGImage into a new context, applying the transform
     // calculated above.
-    CGContextRef ctx = CGBitmapContextCreate(NULL, (size_t) image.size.width, (size_t) image.size.height,
+    CGContextRef ctx = CGBitmapContextCreate(NULL, (size_t)image.size.width, (size_t) image.size.height,
                                              CGImageGetBitsPerComponent(image.CGImage), 0,
                                              CGImageGetColorSpace(image.CGImage),
                                              CGImageGetBitmapInfo(image.CGImage));
     CGContextConcatCTM(ctx, transform);
+    CGRect rect;
     switch (image.imageOrientation) {
         case UIImageOrientationLeft:
         case UIImageOrientationLeftMirrored:
         case UIImageOrientationRight:
         case UIImageOrientationRightMirrored:
-            // Grr...
-            CGContextDrawImage(ctx, CGRectMake(0,0,image.size.height,image.size.width), image.CGImage);
+            rect = CGRectMake(0,0,image.size.height,image.size.width);
             break;
-            
         default:
-            CGContextDrawImage(ctx, CGRectMake(0,0,image.size.width,image.size.height), image.CGImage);
+            rect = CGRectMake(0,0,image.size.width,image.size.height);
             break;
     }
+    CGContextDrawImage(ctx, rect, image.CGImage);
     
     // And now we just create a new UIImage from the drawing context
     CGImageRef cgimg = CGBitmapContextCreateImage(ctx);
     UIImage *img = [UIImage imageWithCGImage:cgimg];
     CGContextRelease(ctx);
     CGImageRelease(cgimg);
     return img;
-    
 }
 
--(NSString*) filePathForResourceName:(NSString*) name withExtension:(NSString*) extension {
+- (NSString*)filePathForResourceName:(NSString*)name withExtension:(NSString*)extension {
     NSString* file_path = [[NSBundle mainBundle] pathForResource:name ofType:extension];
-    if (file_path == NULL) {
+    if (!file_path) {
         LOG(FATAL) << "Couldn't find '" << [name UTF8String] << "."
         << [extension UTF8String] << "' in bundle.";
     }
     return file_path;
 }
 
--(NSString*) runInferenceOnImage:(UIImage *) image {
+- (NSString*)runInferenceOnImage:(UIImage *)image {
     std::string graph;
     const int num_threads = 1;
     std::string input_layer_type = "float";
     std::vector<int> sizes = {1, 224, 224, 3};
     
     NSString* graph_path = [self filePathForResourceName:@"mobilenet_v1_1.0_224" withExtension: @"tflite"];
     
-    std::unique_ptr<tflite::FlatBufferModel> model(tflite::FlatBufferModel::BuildFromFile([graph_path UTF8String]));
+    std::unique_ptr<tflite::FlatBufferModel> model(tflite::FlatBufferModel::BuildFromFile([graph_path fileSystemRepresentation]));
     if (!model) {
         LOG(FATAL) << "Failed to mmap model " << graph;
```

Comment:
Use an NSAssert here... as it is, graph is going to be useless info.. you probably want graph_path.